### PR TITLE
Added a missing / to pr2-brand and changed postinst branding to defaults

### PR DESCRIPTION
**Pull request by archielee**
@TheDash 

Defaults are from the pr2-brand script where the default basestation IP is 10.68.255.1 and there is no default basestation hostname so I just went with "basestation"

### CODE REVIEW

**Comment by archielee** [comment]
This also assumes that the PR2 has not been branded before, running it if the PR2 has a different hostname already will kill networking

File: debian/postinst [hunk 1]
```diff
@@ -46,11 +46,11 @@ try:
...
   print 'robot'
 """`
-        C1_NAME=`cat /etc/hostname`
-        C2_NAME=`cat /unionfs/overlay/etc/hostname`
+        C1_NAME="c1"
+        C2_NAME="c2"
         NUC_NAME="pr2-head"
-        BASESTATION_IP=`cat /tmp/pr2-network/BASESTATION_IP`
-        BASESTATION_HOSTNAME=`cat /tmp/pr2-network/BASESTATION_HOSTNAME`
+        BASESTATION_IP="10.68.255.1"
```
Comment (TheDash) [inline debian/postinst]: This is not the basestation IP... this is a variable that is to be set by the user, same with C1 and C2.

File: debian/postinst [hunk 2]
```diff
@@ -46,11 +46,11 @@ try:
...
+        BASESTATION_HOSTNAME="basestation"
```
Comment (TheDash) [inline debian/postinst]: This is to be set by the user.

File: root/usr/sbin/pr2-brand [hunk 3]
```diff
@@ -90,7 +90,7 @@ s/10\.68\.0\.91(\s+\S+){1,2}/10\.68\.0\.91 c1-esms ${MASTER_NAME}-esms/;
 s/10\.68\.0\.2(\s+\S+){2,4}/10\.68\.0\.2 ${SLAVE_NAME} c2 c2-lan0 ${SLAVE_NAME}-lan0/;
 s/10\.68\.0\.12(\s+\S+){1,2}/10\.68\.0\.12 c2-lan1 ${SLAVE_NAME}-lan1/;
 s/10\.68\.0\.92(\s+\S+){1,2}/10\.68\.0\.92 c2-esms ${SLAVE_NAME}-esms/;
-s/10\.68\.0\.10(\s+\S+){2,4}/10\.68\.0\.10 ${NUC_NAME} pr2-head;
+s/10\.68\.0\.10(\s+\S+){2,4}/10\.68\.0\.10 ${NUC_NAME} pr2-head/;
```
Comment (TheDash) [inline root/usr/sbin/pr2-brand]: Looks good. Without this the hosts file doesn't get generated and everything is ******** from there.

**Comment by TheDash** [comment]
Please remove other changes other than the "/" fix, and make another PR to merge the pr2-brand stuff out.

**Pull request closed without merging**